package.json

```diff
@@ -15,13 +15,11 @@
     "escodegen": "^1.11.1"
   },
   "devDependencies": {
-    "eslint": "^6.0.0",
-    "eslint-config-apostrophe": "^3.1.0",
-    "eslint-config-standard": "^11.0.0-beta.0",
-    "eslint-plugin-import": "^2.16.0",
-    "eslint-plugin-node": "^5.2.1",
-    "eslint-plugin-promise": "^3.6.0",
-    "eslint-plugin-standard": "^3.0.1"
+    "eslint": "^8.0.0",
```

Comment:
Our config may not work with v8

Reply:
`"eslint": "^7.32.0",` and a fresh install should do it. I realize there are warnings for v7.